includes/class-starter-content.php

```diff
@@ -359,7 +359,8 @@ private static function add_featured_image( $post_id, $post_index ) {
 			require_once ABSPATH . 'wp-admin/includes/image.php';
 		}
 
-		$url = self::is_e2e() ? 'https://picsum.photos/id/' . $post_index . '/1200/800' : 'https://picsum.photos/1200/800';
+		// Use same image everywhere for e2e.
+		$url = self::is_e2e() ? 'https://picsum.photos/id/424/1200/800' : 'https://picsum.photos/1200/800';
```

Comment:
This is a very nice picture!

<img src="https://i.picsum.photos/id/424/1200/800.jpg" />

Reply:
My first shot was [id 42](https://picsum.photos/id/42/1200/800), but it was not the answer to life, universe, and everything. This is is closer to it IMO.